changelog.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## NEXT
+
+- Fixed a bug that would allow the addition of b--a and b->a edges to a graph with an existing undirected a--b edge in
+  the `cai_causal_graph.causal_graph.CausalGraph._set_edge` method.
+  - Added the `CausalGraphsError.ReverseEdgeExistsError` exception, which distinguishes errors arising from the
```

Comment:
Think it should be `CausalGraphError.ReverseEdgeExistsError` instead (you had one extra s in there)